ui/src/app/components/pdf-viewer.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+import {Document, Page} from 'react-pdf';
+
+import {PdfViewer, Props} from 'app/components/pdf-viewer';
+
+let props = {
+  windowSize: {width: 1700, height: 0},
+  pdfPath: '/assets/documents/fake-document-path.pdf'
```

Comment:
Could you have a `/test/assets/documents/pdf1.pdf` or something and test with a real doc?